hydroflow_lang/src/graph/graph_write.rs

```diff
@@ -196,8 +196,10 @@ where
                 self.link_count,
                 match (delay_type, lattice_flow_type) {
                     (None, None) => unreachable!(),
-                    (Some(_), _) => "red",
-                    (None, Some(_)) => "#060",
+                    (Some(DelayType::Stratum), _)
+                    | (Some(DelayType::Tick), _)
+                    | (Some(DelayType::TickLazy), _) => "red",
+                    (Some(DelayType::MonotoneAccum), _) | (None, Some(_)) => "#060",
                 }
```

Comment:
For `(Some(DelayType::MonotoneAccum), _)` I would assume `lattice_flow_type` should be set to something usually. No changes needed here

Reply:
Something to revisit when we go over flow props more carefully.